src/content/posts/2024/07/16/empower-teams-to-build-better-apps-faster.mdx

```diff
@@ -0,0 +1,75 @@
+
+## The lifecycle of an app
+
+The toolchain will reflect the lifecycle of apps, from their creation to their delivery to users. The CLI, the dashboard, and our documentation will all mirror this lifecycle. We believe it’s crucial for developers to feel comfortable understanding and using the toolchain. Long-term, we aim for Tuist’s interface, which is very client-centric today, to evolve as follows:
+
+- **tuist init:** Create a new project from community or Tuist-provided templates.
+- **tuist build:** Build my project reliably and quickly, collecting insights.
+- **tuist test:** Test my project reliably and quickly, collecting insights such as test coverage, performance, and flakiness.
+- **tuist lint:** Validate the project or the artifacts generated from builds.
+- **tuist share:** Share this app with my team or external people for feedback.
+- **tuist workflow:** Run a Swift-based workflow locally, or manage the ones running remotely.
+- **tuist release:** Trigger a release of my app to the App Store or any other distribution channel.
+
+Drawing inspiration from the [Ruby](https://www.ruby-lang.org/en/) and [Ruby on Rails](https://rubyonrails.org/) communities, from which we have learned a lot, we will design the interface of Tuist to optimize for joy and invest in Swift extensibility to foster a vibrant ecosystem of plugins and integrations. More on this in the coming months.
+
+## Embracing openness
+
+The best organizations that we aspire to be like have one thing in common: *openness* (e.g., [Supabase](https://supabase.com/), [GitLab](https://gitlab.com), [PostHog](https://posthog.com)). Being open is what has brought us to where we are. It’s how we built trust with our users, created a foundation for diverse ideas, fostered collaboration, and, without a doubt, developed high-quality software that lasts and solves real problems. We are committed to doubling down on this value.
+
+We are working on establishing a path and timeline to open-source everything we do without compromising our ability to build a sustainable business. We are documenting how we are building the company in a [public handbook](https://handbook.tuist.io) that is available for everyone to read. We believe that having **a handbook will enable a frictionless and asynchronous remote work environment, attracting the best talent from around the world.** As first-time founders on this journey, we also aim to inspire others to start their own companies and share their learnings with the world.
+
+We will also **commoditize some pieces of technology** that we’ve noticed are common across many organizations to help elevate innovation at different layers of the stack. We believe that by doing so, we can help organizations focus on what differentiates them rather than on undifferentiated heavy lifting. Expect a handful of open-source projects from us in the coming months.
```

Comment:
```suggestion
We will also **commoditize some pieces of technology** that we’ve noticed are common across many organizations to help elevate innovation at different layers of the stack. We believe that by doing so, we can help organizations focus on what differentiates them rather than on focusing on zero-sum game. Expect a handful of open-source projects from us in the coming months.
```

Not sure if this is the best way to rephrase, but `undifferentiated heavy lifting` is too vague imho.